docs/70-jobs.md

```diff
@@ -0,0 +1,172 @@
+
+
+Below we will define a simple, functioning job definition.
+```yaml
+# Job Configuration
+
+job:
+  id: test-job
+  cluster_configuration:
+    vm_size: standard_f2
+    size: 3
+ 
+  applications:
+    - name: pipy100
+      application: /path/to/pi.py
+      application_args: 
+        - 100
+    - name: pipy200
+      application: /path/to/pi.py
+      application_args: 
+        - 200
+```
+Once submitted, this Job will run two applications, pipy100 and pipy200, on an automatically provisioned Cluster with 3 dedicated Standard_f2 size Azure VMs. Immediately after both pipy100 and pipy200 have completed the Cluster will be destroyed. Application logs will be persisted and available.
+
+### Commands
+Submit a Spark Job:
+
+```sh
+aztk spark job submit --id <your_job_id> --job-conf </path/to/job.yaml>
+```
+
+NOTE: The Job id (`--id`) can only contain alphanumeric characters including hyphens and underscores, and cannot contain more than 64 characters. Each cluster **must** have a unique cluster id.
+
+
+#### Low priority nodes
+You can create your Job with [low-priority](https://docs.microsoft.com/en-us/azure/batch/batch-low-pri-vms) VMs at an 80% discount by using `--size-low-pri` instead of `--size`. Note that these are great for experimental use, but can be taken away at any time. We recommend against this option when doing long running jobs or for critical workloads.
+
+
+### Listing Jobs
+You can list all Jobs currently running in your account by running
+
+```sh
+aztk spark job list
+```
+
+
+### Viewing a Job
+To view details about a particular Job, run:
+
+```sh
+aztk spark job get --id <your_job_id>
+```
+
+For example here Job 'pipy' has 2 applications which have already completed.
+
+```sh
+Job             pipy
+------------------------------------------
+State:                              | completed
+Transition Time:                    | 21:29PM 11/12/17
+
+Applications                        | State          | Transition Time
+------------------------------------|----------------|-----------------
+pipy100                             | completed      | 21:25PM 11/12/17
+pipy200                             | completed      | 21:24PM 11/12/17
+```
+
+
+### Deleting a Job
+To delete a Job run:
+
+```sh
+aztk spark job delete --id <your_job_id>
+```
+
+__You are only charged for the job while it is active, Jobs handle provisioning and destorying infrastructure, so you are only charged for the time that your applications are running.__
+
+
+### Stopping a Job
+To stop a Job run:
+
+```sh
+aztk spark job stop --id <your_job_id>
+```
+Stopping a Job will end any currently running Applications and will prevent any new Applications from running.
+
+
+### Get information about a Job's Application
+To get information about a Job's Application:
+
+```sh
+aztk spark job get-app --id <your_job_id> --name <your_application_name>
+```
+
+
+### Getting a Job's Application's log
```

Comment:
Is there a way to delete application logs? If not we should have an issue for that after this ships.

Reply:
Not currently. Will open an issue. Cleaning up storage should also happen as a part of cluster deletion for interactive mode like we have talked about before.